apps/web/src/components/data-table/monitor/data-table-row-actions.tsx

```diff
@@ -156,21 +171,38 @@ export function DataTableRowActions<TData>({
         <AlertDialogHeader>
           <AlertDialogTitle>Are you absolutely sure?</AlertDialogTitle>
           <AlertDialogDescription>
-            This action cannot be undone. This will permanently delete the
-            monitor.
+            {alertType === "delete"
+              ? "This action cannot be undone. This will permanently delete the monitor."
+              : "This action cannot be undone. This will clone the monitor."}
           </AlertDialogDescription>
         </AlertDialogHeader>
         <AlertDialogFooter>
           <AlertDialogCancel>Cancel</AlertDialogCancel>
           <AlertDialogAction
             onClick={(e) => {
               e.preventDefault();
-              onDelete();
+              if (alertType === "delete") onDelete();
+              else if (alertType === "clone") onClone();
             }}
             disabled={isPending}
-            className="bg-destructive text-destructive-foreground hover:bg-destructive/90"
+            className={cn({
+              "bg-destructive text-destructive-foreground hover:bg-destructive/90":
+                alertType === "delete",
+              "bg-primary text-primary-foreground hover:bg-primary/90":
+                alertType === "clone",
+            })}
```

Comment:
How about using `asChild` and use the

```ts
<Button variant={x ? "default" : "destructive"}>...</Button>
```

Reply:
Done

Reply:
@mxkaske , For some reason  `variant={alerts[alertType].action.variant}` is not working so using ```className={cn(buttonVariants({ variant: alerts[alertType].action.variant }))}```  instead